applications/matter_bridge/CMakeLists.txt

```diff
@@ -47,13 +47,13 @@ target_sources(app PRIVATE
     ${COMMON_ROOT}/src/bridge/matter_bridged_device.cpp
     ${COMMON_ROOT}/src/bridge/bridge_storage_manager.cpp
     ${COMMON_ROOT}/src/bridge/bridged_device_data_provider.cpp
-    ${COMMON_ROOT}/src/bridge/binding_handler.cpp
+    ${COMMON_ROOT}/src/binding/binding_handler.cpp
     src/zap-generated/IMClusterCommandHandler.cpp
     src/zap-generated/callback-stub.cpp
-    ${COMMON_ROOT}/src/led_widget.cpp
-    ${COMMON_ROOT}/src/persistent_storage_util.cpp
-    ${COMMON_ROOT}/src/task_executor.cpp
-    ${COMMON_ROOT}/src/board.cpp
+    ${COMMON_ROOT}/src/led/led_widget.cpp
```

Comment:
This commit mixes cleanup of the `common` directory and utilization of the common binding handler in the bridge application. I don't know what's other opinion but to me it should be fixed to keep the git history clean (@kkasperczyk-no, @ArekBalysNordic ?). I also suggest to change the commit message: "...Implement common binding manager" to "...Use common binding handler"

Reply:
EDIT: Ok I see Arek shares the same concerns I do, so this definitely must be fixed :)

Reply:
Yeah, I agree with Kamil, as far as we as Nordic developers are aware of that because sometimes we checkout between different commits.

Reply:
Sorry for that mess, by mistake I've added a whole bridge to one commit. It should be cleaner right now